archived/star-rx72n-firmware/tests/test_rx_drv8243_spi.c

```
@@ -0,0 +1,1187 @@
+  s_fault_pin  = 0;
+
+  /* Reset RSPI mock */
+  mock_rspi_init(NULL);
+}
+
+/* mock_set_fault is currently unused but kept for future fault simulation tests */
+static void mock_set_fault(bool active) __attribute__((unused));
+static void mock_set_fault(bool active)
+{
+  s_fault_active = active;
+  if (active) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)